msal/oauth2cli/oidc.py

```diff
@@ -44,10 +44,11 @@ def decode_id_token(id_token, client_id=None, issuer=None, nonce=None, now=None)
     err = None  # https://openid.net/specs/openid-connect-core-1_0.html#IDTokenValidation
     _now = int(now or time.time())
     skew = 120  # 2 minutes
+    TIME_SUGGESTION = "Make sure your computer's time is correctly synchronized."
```

Comment:
Perhaps we also need to mention time zone? The time on the right bottom corner may seem correct, but the time zone may be wrong.

![image](https://user-images.githubusercontent.com/4003950/149461683-62fc69bb-b29d-4984-813c-4b689d780792.png)

Reply:
Good point. I'll change that line to "Make sure your computer's time **(including timezone)** is correctly synchronized.".